LiteCore/Support/EncryptedStream.cc

```diff
@@ -19,7 +19,7 @@
 #include "SecureRandomize.hh"
 #include "SecureSymmetricCrypto.hh"
 #include "Endian.hh"
-
+#include <minmax.h>
```

Comment:
For some reason Windows started complaining about the lack of a `MIN` definition (or possibly `MAX`).  Maybe due to a header include order change?